xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -125,7 +125,7 @@ void CPVRRecordings::GetSubDirectories(const CPVRRecordingsPath &recParentPath,
       pFileItem->m_dateTime = current->RecordingTimeAsLocalTime();
 
       // Assume all folders are watched, we'll change the overlay later
-      pFileItem->SetOverlayImage(CGUIListItem::ICON_OVERLAY_WATCHED, false);
+      pFileItem->SetOverlayImage(CGUIListItem::ICON_OVERLAY_WATCHED);
```

Comment:
> Indeed this is a noop, but to set an overlay, SetOverlayImage() should be called without the 2nd parameter... 

 This API is crap, if you ask me. But if you insist in removing the parameter, well, so shall it be. ;-)